### ISSUE TITLE
[Bug] The training set is not shuffled between epochs when user sets shuffle=True

### ISSUE THREAD

**Issue by SiriusHou** [ant-research/EasyTemporalPointProcess#54]
[Here,](https://github.com/ant-research/EasyTemporalPointProcess/blob/e27440645e2278a6d20aa0d0bdec3ee53aaf91d5/easy_tpp/config_factory/runner_config.py#L95) you not only disable shuffling for the validation set but also for the training set.

**Comment by iLampard** [comment]
Hi, 

I have remove the 'ensure_valid_config' function and enable shuffle in both train and valid data loaders. 

The code has been pushed into main branch.
